map1.yml

```diff
@@ -0,0 +1,9 @@
+- connected: true
+  id: 000440203861
+  platform: efm32pg_stk3402a
+  product: J-Link OB
+  runner: jlink
+  serial: /dev/ttyUSB0
+  baud: '9600'
+  fixtures:
+    - gpio_loopback
```

Comment:
Was this file meant to be included?

Reply:
I included this file because I still am unsure how to document the baud entry. I recognize that the map.yml files are unique to each user, but this simply serves as a placeholder for me to ask where this should be documented.

Reply:
it should be removed, there is a section in the docs where you can describe this. (`doc/guides/test/twister.rst`)

Reply:
Thanks! Should that be included with the twister PR or on its own?